Updated_GM_Code.R

```diff
@@ -0,0 +1,46 @@
+library(geomorph)
```

Comment:
For some reason, it took forever for this library to load for me... But I think that's a problem on my end